### DISCUSSION TITLE
Unconfirmed UTXO (Remote Force Close)

### DISCUSSION THREAD

**Discussion by iamn3mo** [Ride-The-Lightning/RTL#1283] [Troubleshooting Questions] [answered]
Hi,
some 8 weeks ago a remote force close happened that i didn't notice until today. In the "Lightning" panel it shows a settled balance of about 2700000 sats. I can see the confirmed closing transaction in the blockchain and can follow the funds to an unspent address/UTXO.

However, in the "On-chain" panel of RTL the 2700000 are shown as unconfirmed. Neither the shown Transaction ID nor the adress of that UTXO can be found in the blockchain or mempool. Any guess what happened here, and how do i get the funds back?

I am using RTL bundled with BTCPayServer. I did update the server und restarted it right now, with no change. Help is highly apperciated.

Edit:
I do have an idea where the issue originates from. The implemantion i am using is LND in combination with a pruned node (which isn't recommended) and the closed channel was about 1 year old. Might this be the issue?

Also, in the "On-chain" panel, the unconfirmed UTXO is labeled as "0:sweep" , which is strange.

**Comment by saubyk** [comment, marked as answer]
hi @nemoBTC the `0:sweep` label is applied to sweeps which are used to convert local funds from force closures to wallet spendable outputs. I am not sure why the UTXO shown in RTL is not trackable anywhere. But you if can see the closing txn on the blockchain, you should be able to track it to the spendable output for your wallet.

You do have unique scenario in that you're running with a pruned node. You have the following options to get your UTXOs back in your wallet:
Option 1:
Upgrade your node to full node and rescan the wallet of your LND node. This is a difficult option because rescanning requires restarting the LND node with specific config setting, which may be difficult to do given that you're running with BTC Payserver.

Option 2:
Get the 24 word key mnemonic for your LND node and use that to recover your onchain wallet. Getting your key phrase also is a difficult task given that you're running BTC Payserver.

You would need help with the BTC Payserver support to execute both options, but operationally option 2 might be easier to work with.

**Comment by iamn3mo** [comment]
Hi @saubyk thank you very much for your response. I am running my own instance of BTCPayServer and have full acces and control, so i should be able to perform both options.
